src/extensions/default/bramble/main.js

```diff
@@ -326,7 +265,7 @@ define(function (require, exports, module) {
                     fn: data.fn,
                     value: typeof value !== "object" ? value : undefined
                 }), "*");
-            });
+            }, false);
```

Comment:
It's an old habit, see https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener and the third arg.